pages/report-hazard/index.html

```diff
@@ -25,6 +25,8 @@
     <script src="../../modules/leaflet/leaflet.js"></script>
     <!-- Global Styles -->
     <link rel="stylesheet" href="../../assets/css/_global.css" />
+    <!-- White transition Styles -->
+    <link rel="stylesheet" href="../../assets/css/white-transition.css" />
```

Comment:
move this import to inside the style.css

Reply:
Done